### ISSUE TITLE
Button [new] doesn't respond to mouse click.

### ISSUE THREAD

**Issue by gryf** [gryf/tabbedalt#12]
I received a bug report by email from one of the users, who have in settings:

```
URxvt.tabbed.new-button:	true
```

button `[NEW]` shows up, but fail to respond on mouse click with error message:

```
urxvt: Undefined subroutine &main::-1 called at /home/gryf/.urxvt/ext/tabbed line 364.
```

**Comment by gryf** [comment]
Fixed in 6fc444851fa08fe6a63765530172bc44231efa68.

**Comment by kdato49** [comment]
Thanks for fixing this so quickly, it works. I think changing the name of the plug-in to avoid confusion was a good idea.
